media-sound/pnmixer/pnmixer-0.7.2-r1.ebuild

```diff
@@ -0,0 +1,59 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_P="${PN}-v${PV}"
+inherit cmake xdg-utils
```

Comment:
You not joining the xdg bandwagon?

Reply:
`xdg.eclass` always contains the potential for future trouble with `cmake.eclass` because both export `src_prepare` and inherit order then suddenly becomes a factor. When it's not my own ebuilds I don't enforce one or the other method.